tests/lib/cli.js

```diff
@@ -163,162 +119,113 @@ describe("cli", () => {
 			sh.rm("-r", fixtureDir);
 		});
 
-		["eslintrc", "flat"].forEach(configType => {
-			const useFlatConfig = configType === "flat";
-			const ActiveESLint = configType === "flat" ? ESLint : LegacyESLint;
-
-			describe("execute()", () => {
-				it(`should return error when text with incorrect quotes is passed as argument with configType:${configType}`, async () => {
-					const flag = useFlatConfig
-						? "--no-config-lookup"
-						: "--no-eslintrc";
-					const configFile = getFixturePath(
-						"configurations",
-						"quotes-error.js",
-					);
-					const result = await cli.execute(
-						`${flag} -c ${configFile} --stdin --stdin-filename foo.js`,
-						"var foo = 'bar';",
-						useFlatConfig,
-					);
-
-					assert.strictEqual(result, 1);
-				});
-
-				it(`should not print debug info when passed the empty string as text with configType:${configType}`, async () => {
-					const flag = useFlatConfig
-						? "--no-config-lookup"
-						: "--no-eslintrc";
-					const result = await cli.execute(
-						[
-							"argv0",
-							"argv1",
-							"--stdin",
-							flag,
-							"--stdin-filename",
-							"foo.js",
-						],
-						"",
-						useFlatConfig,
-					);
-
-					assert.strictEqual(result, 0);
-					assert.isTrue(log.info.notCalled);
-				});
-
-				it(`should exit with console error when passed unsupported arguments with configType:${configType}`, async () => {
-					const filePath = getFixturePath("files");
-					const result = await cli.execute(
-						`--blah --another ${filePath}`,
-						null,
-						useFlatConfig,
-					);
-
-					assert.strictEqual(result, 2);
-				});
+		describe("execute()", () => {
+			it(`should return error when text with incorrect quotes is passed as argument`, async () => {
+				const configFile = getFixturePath(
+					"configurations",
+					"quotes-error.js",
+				);
+				const result = await cli.execute(
+					`--no-config-lookup -c ${configFile} --stdin --stdin-filename foo.js`,
+					"var foo = 'bar';",
+				);
+
+				assert.strictEqual(result, 1);
 			});
 
-			describe("flat config", () => {
```

Comment:
These tests seem unnecessary now that the CLI only supports flat config, so I removed them rather than fixing them. If we prefer to keep them, I can recreate them with different assertions.